app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -133,4 +155,13 @@ def build_connect_params(options)
 
     connect_params
   end
+
+  def vm_host_handle(vm, options)
+    require 'Scvmm/miq_scvmm_vm_ssa_info'
+    raise "no credentials defined in options #{options}" if self.missing_credentials?(options[:auth_type])
+    user = options[:user] || authentication_userid(options[:auth_type])
+    pass = options[:pass] || authentication_password(options[:auth_type])
+
+    MiqScvmmVmSSAInfo.new(vm.host.hostname, user, pass)
```

Comment:
renamed vm_host_handle to host_handle, moved it to microsoft/infra_manager/host.rb, removed first (unnecessary) argument, and changed the first parameter to MiqScvmmVmSSAInfo.new to hostname.